hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -119,8 +119,8 @@ def _collect_file(self, parent_item, path, frame_sequence=False):
 
         # get info for the extension
         item_info = self._get_item_info(path)
-        item_type = item_info["item_type"]
-        type_display = item_info["type_display"]
+        item_type = item_info["item_type"].encode("utf-8")
```

Comment:
I'm surprised this actually has any affect since `item_type` and `type_display` come from the `COMMON_FILE_INFO` dictionary at the top of this file. 

The exception is if the `category_type` is extracted from the mimetypes (Line `293`). Perhaps that was the reason for the issue? Maybe we should ensure `utf-8` encoding on line `297` instead?

Or perhaps the real issue is the extraction of `category_type` with a filename that includes unicode characters (Line `293`)?